### PR TITLE
Feature/initial value

### PR DESCRIPTION
**Pull request by kmruiz**
@carlosvillu 
@davecarter 

This enables the sui-autocompleted to get the initial value of the input text.

### CODE REVIEW

File: src/sui-autocompleted/index.jsx [hunk 1]
```diff
@@ -95,5 +95,6 @@ Autocompleted.propTypes = {
   placeholder: React.PropTypes.string,
   suggests: React.PropTypes.array.isRequired,
   handleChange: React.PropTypes.func.isRequired,
-  handleSelect: React.PropTypes.func.isRequired
+  handleSelect: React.PropTypes.func.isRequired,
+  initialValue: React.PropTypes.string.isRequired
```
Comment (davecarter) [inline src/sui-autocompleted/index.jsx]: `initialValue` should'n be required IMHO

Comment (kmruiz) [inline src/sui-autocompleted/index.jsx]: :+1: I'll change it now

**Comment by davecarter** [comment]
👍  Perfect!

**Comment by kmruiz** [comment]
:+1:

**Pull request merged**